### ISSUE TITLE
Keep items in local IndexedDb after signOut

### ISSUE THREAD

**Issue by PiTomar** [hoodiehq/hoodie#889]
Hello, when I’m connected/synced to the hoodie backend and do a signOut, all local items are removed from the indexedDb. Is there a way to keep them even after a signOut?
(I would like to keep them offline and just resync on next connection)

Thanks nad kind regards.

**Comment by gr2m** [comment]
I agree that it would be nice to keep the data locally after signOut, but it would need to be safely encrypted for security reasons and then decrypted on login. we have an open issue on that somewhere. It's currently not possible, because it's not trivial.

**Comment by PiTomar** [comment]
Could this encryption be done with something like the hoodie-plugin-store-crypto?https://github.com/Terreii/hoodie-plugin-store-crypto

**Comment by gr2m** [comment]
Maybe, I didn't look into it myself
